api/SpeechSynthesis.json

```
@@ -239,7 +239,8 @@
               "version_added": "33"
             },
             "chrome_android": {
-              "version_added": "33"
+              "version_added": "33",
+              "notes": "In Android, <code>pause()</code> ends the current utterance.  As such, <code>pause()</code> behaves just like <code>cancel()</code>."
```

A little bit of editing here. As a tech writer, I have developed a strong aversion to the word "just", please excuse me

```suggestion
              "notes": "In Android, <code>pause()</code> ends the current utterance. <code>pause()</code> behaves the same as <code>cancel()</code>."
```